Submitted by: Jeremy Huntwork (jhuntwork at linuxfromscratch dot org)
Date: 2005-09-15
Initial Package Version: 1.0.14
Origin: LFS LiveCD
Upstream Status: Submitted
Description: Fixes the Makefile used for compiling unionfs into the kernel
	     instead of as a module.

diff -Naur unionfs-1.0.14.orig/Makefile.kernel unionfs-1.0.14/Makefile.kernel
--- unionfs-1.0.14.orig/Makefile.kernel	2005-09-15 14:55:43.000000000 -0400
+++ unionfs-1.0.14/Makefile.kernel	2005-09-15 20:15:16.683211384 -0400
@@ -1,7 +1,7 @@
 UNIONFS_VERSION  = 1.0.14
-EXTRACFLAGS+=-DUNIONFS_VERSION=\"${UNIONFS_VERSION}\"
+EXTRA_CFLAGS+=-DUNIONFS_VERSION=\"$(UNIONFS_VERSION)\"
 # This will disable debugging support
-# EXTRACFLAGS+=-DUNIONFS_NDEBUG
+# EXTRA_CFLAGS+=-DUNIONFS_NDEBUG
 
 obj-$(CONFIG_UNION_FS) += unionfs.o
 
